quicfire_tools/inputs.py

```diff
@@ -117,6 +118,27 @@ def from_dict(cls, data: dict):
         return cls(**data)
 
 
+class OutputTimes(BaseModel):
+    out_time_fire: PositiveInt
+    out_time_wind: PositiveInt
+    out_time_emis_rad: PositiveInt
+    out_time_wind_avg: PositiveInt
+
+    def __str__(self):
+        lines = (
+            f"{self.out_time_fire}\t! After how many fire time steps to print out fire-related files (excluding emissions and radiation)\n"
+            f"{self.out_time_wind}\t! After how many quic updates to print out wind-related files\n"
+            f"{self.out_time_emis_rad}\t! After how many fire time steps to average emissions and radiation\n"
+            f"{self.out_time_wind_avg}\t! After how many quic updates to print out averaged wind-related files")
+        return lines
+
+
+class FuelParams(BaseModel):
```

Comment:
Same as OutputTimes. What does using a class here instead of attributes to QUIC_Fire buy us? Lets talk about this on Tuesday.

Reply:
Oh I see you're asking about FuelParams. Sure let's chat